app/views/RoomView/index.tsx

```diff
@@ -621,18 +621,20 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 			),
 			headerRight: () => (
 				<RightButtons
-					rid={rid}
+					rid={rid!}
```

Comment:
No bangs.
This might break the app on tablets, for example, since it renders RoomView without `rid`.